### ISSUE TITLE
Should update to match the new spec

### ISSUE THREAD

**Issue by zcorpan** [weblinc/img-srcset#4]
It looks like this polyfill implements the old spec for srcset, where 'w' descriptors meant viewport max-width. The current spec makes 'w' descriptors mean width of the image, to be used together with the sizes attribute. The functionality for the old 'w' is available by using the picture element.

**Comment by booleanbetrayal** [comment]
@zcorpan - Unless I'm mistaken, the polyfill seems correct. According to the [latest working spec](http://drafts.htmlwg.org/srcset/w3c-srcset/#additions-to-the-img-element) (Nov 13 2014): 

> To specify an image, give first a URL, then one or more descriptors of the form 100w or 2x, where "100w" means "**maximum viewport width** of 100 CSS pixels" and "2x" means "maximum pixel density of 2 device pixels per CSS pixel".

**Comment by zcorpan** [comment]
Aaargh! That's the old spec. -_- We tried to kill all copies but it seems W3C is good at producing many copies scattered around so it's hard to catch all of them. Also the spec's date is a lie.

You want to look at https://html.spec.whatwg.org/multipage/embedded-content.html#introduction-3

cheers

**Comment by knightdr** [comment]
Thanks @zcorpan and @booleanbetrayal for your comments.

@zcorpan, would you say borismus/srcset-polyfill is the definitive polyfill for this functionality? If so, I'd like to put effort into fixing its issues rather than maintaining this project which may/will have its own issues. I appreciate any feedback you have to offer.

**Comment by zcorpan** [comment]
No, I just wanted to point to my comment above instead of copy-pasting it.

If you are looking to contribute to a different project that is further along, I suggest checking https://github.com/scottjehl/picturefill/issues/374

**Comment by Wilto** [comment]
We’d love to have the help over at Picturefill, if you’re interested!

**Comment by knightdr** [comment]
Thanks @Wilto, I've been watching the project for a long time. I contributed early on before version 2 but haven't since, just because it looked like there was good movement and a lot changing. I didn't want to add extra noise.
